lint-rules/build.gradle

```diff
@@ -34,4 +34,10 @@ java {
 
 tasks.register('jvm_checks') {
     dependsOn 'test'
+}
+
+jar {
```

Comment:
why this change? I don't see any description about it. Does this still make the `com.duckduckgo.lint.ui.DuckDuckGoDesignSystemRegistry` work?

Reply:
It's still WIP so incomplete. Design registry not wired in yet. 

I couldn't find the right syntax to add both registries in. It might be possible with the right syntax, or it might be you can only register one and we'd have to combine the registries. Not sure yet. 

Let's discuss more why the PR at all next week, but TL;DR is that our custom rules aren't running at the moment; only their tests execute but not the lint rules themselves.